source/get-started/containers/container-images.rst

```diff
@@ -0,0 +1,105 @@
+.. _container-images:
+
+|CL-ATTR| container images
+##########################
+
+|CL| can run inside of a container on top of any operating system as long as
+it is hosting a containerized environment, such as Docker* or Kubernetes*.
+Ready-to-use |CL| container images are available on Docker Hub*. 
+
+A |CL| base image is available for standalone use. Additionally, variations of
+popular application images built from the |CL| base image are also published.
+
+All |CL| container images can be `browsed on the Docker Hub* website
+<https://hub.docker.com/search?q=clearlinux&type=image>`_. The corresponding
+Dockerfiles used to build |CL| container images are `on GitHub
+<https://github.com/clearlinux/dockerfiles>`_.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Features
+********
+
+|CL| container images have:
+
+* **Optimized software:** images have software and libraries compiled with the
+  latest optimizations, including Intel® Advanced Vector Extensions AVX-512
```

Comment:
images include the latest optimizations of Intel Advanced Vector Extensions... (Intel AVX2-512).

Reply:
I think these bulleted items work better if they replaced current bulleted list here: https://clearlinux.org/downloads/containers.  Then, you can simply make a link on this page to the "containers" page. 
I feel that the Docs should cover the what and how more than the why.

Reply:
The purposed wording of the first bullet would be inaccurate. 
The list on the page you linked is for using |CL| as a container host where as this is using |CL| as a container. 

My plan is the rework the page you linked after this is published.

Reply:
I'm open to reworking https://clearlinux.org/downloads/containers.  I think this page should be revised to include both `|CL| as a container host` and `|CL|  as a container`. I think it's more appropriate to explain all the benefits of these two options on [the containers marketing page](https://clearlinux.org/downloads/container), and keep our docs uncluttered. Then  we can add a hyperlink on this document to the marketing page. Let's discuss your plan in person for revisions to the marketing page. I wasn't aware you intended to do this too.

Reply:
I've pushed an update moving the features section out and linking to https://clearlinux.org/downloads/containers. We'll update that page and the DockerHub overview pages separately as we discussed.

Reply:
Great. Thanks.